src/mcts/search.cc

```diff
@@ -682,7 +682,25 @@ void SearchWorker::ExtendNode(Node* node) {
       return;
     }
   }
-
+  
+  //WDL probing
+  if(board.our + board.their <= SyzygyTablebase::max_cardinality && board.GetNoCapturePly == 0
+     && board.no_legal_castle()) {
+    SyzygyTablebase::ProbeState* result;
+    WDLScore wdl = probe_wdl(board.pos, &result);
```

Comment:
probe_wdl is a method on a SyzygyTablebase instance - so instance->probe_wdl - once you have such an instance available.